ai-service/Dockerfile

```
@@ -54,8 +56,8 @@ FROM python:$PYTHON_BASE
 # curl: needed for healthcheck
 RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)